Feature: `LocalStorage`

edugabriel12 opened the pull request:
### Feature
- Implemented the type class `LocalStorage`, including its `saveHighScore(highScore)` and `readHighScore()` methods and an txt file called `Score.txt` that writes the score within it.

edugabriel12 committed:
Feature: `LocalStorage` object

### Feature
- Implemented the type class `LocalStorage`, including its `saveHighScore(highScore)` and `readHighScore()` methods and an txt file that writes the score within it.

diego-aquino commented on Haskell/Models/Score.txt:
```diff
@@ -0,0 +1 @@
+12
```
This file should be added to `.gitignore`, as it is not meant to be committed, right?

augustonz replied:
Also the `Score.txt` file should be outside of the Models folder

edugabriel12 committed:
Deleting the Score,txt from Models directory

edugabriel12 committed:
Adding the txt file to the project

edugabriel12 committed:
Addind Score.txt to gitignore

edugabriel12 committed:
Delete Score.txt

augustonz reviewed (approved):
Nice

edugabriel12 committed:
Update LocalStorage.hs

Now, the readHighScore method also verifies if the txt file does exist or not. If it does, the flow of execution continues normally, If it doesn´t, the method just returns 0.

edugabriel12 committed:
Merge branch 'Canary' into feature/local-storage

edugabriel12 committed:
Update flap-plp.cabal

Adding `Models.LocalStorage` to the flap-plp.cabal
